### DISCUSSION TITLE
What is issue for next.js

### DISCUSSION THREAD

**Discussion by ChanghyeonYoon** [oven-sh/bun#5794] [General]
Use the `bun` for `Next.js`, is any issue now??
If any issue, I want to contribute for `bun` 😃

**Comment by melroy89** [comment]
https://github.com/oven-sh/bun/issues/5808

https://github.com/oven-sh/bun/issues/4671

https://github.com/oven-sh/bun/issues/4795

https://github.com/oven-sh/bun/issues/5814
